atlp-rwanda/e-comm-team-techsmith-bn
# #184752086 Providing descriptive messages to users

## What does this PR do?

This PR helps provide users with meaningful and descriptive messages when they receive a response.

## Description of Task to be completed?

This was a quick task to fix messages that are being returned to users, especially when their request could not be fulfilled.

## How should this be manually tested?

- Clone this repo using `git clone` [repo](https://github.com/atlp-rwanda/e-comm-team-techsmith-bn)
-  Navigate into the directory by writing `cd e-comm-team-techsmith-bn` in the command line.
-  Run `npm i` to install the project dependencies.
-  Create a file called `.env` in the root directory. In the file, enter all credentials that the applications requires to operate. Please check a file called `.env.example` to find a list of the environment variables you will require.

## What are the relevant pivotal tracker/Trello stories?

- [#184752087](https://www.pivotaltracker.com/story/show/184752087)
-  [#184752086](https://www.pivotaltracker.com/story/show/184752086)
-  [#184752089](https://www.pivotaltracker.com/story/show/184752089)

## Screenshots (if appropriate)

<img width="1440" alt="invalid-email" src="https://user-images.githubusercontent.com/31816352/229729936-f6f81fc4-2e86-47f4-9874-ad3db9918609.png">
<img width="1440" alt="login" src="https://user-images.githubusercontent.com/31816352/229729983-7c3edb41-fa67-467c-bfac-49746800f9ac.png">
<img width="1440" alt="wrong-credentials" src="https://user-images.githubusercontent.com/31816352/229730073-d4f5113e-43b2-45df-9d00-2efe739389df.png">
